### ISSUE TITLE
TO: postinstall on new Centos 6.8 machine fails installing YAML

### ISSUE THREAD

**Issue by dangogh** [Comcast/traffic_control#1583]
During `postinstall`,   I hit the error below.   Tracked down to a new dependency in the YAML perl module (Test::More) that isn't accounted for in the metadata.  This is not part of the carton (cpanfile) install,  it's done in `.../install/bin/build_trafficops_perl_library`.   Recommend adding `cpan Test::More` in that script prior to install of `YAML`.

```
Running make test
PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'blib/lib', 'blib/arch')" t/*.t
t/000-compile-modules.t ... Can't locate Test/More.pm in @INC (@INC contains: /root/.cpan/build/YAML-1.17-Apkbir/blib/lib /root/.cpan/build/YAML-1.17-Apkbir/blib/arch /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at t/000-compile-modules.t line 2.
```

**Comment by dangogh** [comment]
fix in hand..   testing it out now.

**Comment by dangogh** [comment]
Fixed by #1585 on master and #1586 in 1.6.x
